publiccode.yml

```
@@ -0,0 +1,62 @@
+publiccodeYmlVersion: "0"
+name: Integreat CMS
+applicationSuite: Integreat Platform
+url: "https://github.com/digitalfabrik/integreat-cms.git"
+landingURL: "https://integreat-app.de/en/"
+isBasedOn: "https://github.com/django/django.git"
+softwareVersion: "2025.9.0"
+releaseDate: "2025-09-11"
+logo: "https://integreat-app.de/wp-content/uploads/logo-aktuell.svg"
+platforms:
+  - linux
+categories:
+  - content-management
+  - educational-content
+  - translation
+usedBy:
+  - Germany - City of Munich
+  - Germany - City of Dortmund
+developmentStatus: stable
+softwareType: standalone/web
+description:
+  en:
+    shortDescription: The multilingual content management system for the Integreat App
+    longDescription: >
+          The Integreat CMS is a headless content management system for the Integreat App (https://github.com/digitalfabrik/integreat-app).
+          It manages all pages in the same hierarchical structure across all languages. It supports automatic and manual translations.
```

```suggestion
          It manages all pages in the same hierarchical structure across all languages. It supports machine and manual translations.
```
Super duper nit: We try to use machine translation instead of automatic translation in the system as we once noticed that automatic translation is not clear to everyone right away, but I'm sure there are people who would disagree